x-pack/plugins/siem/scripts/gql_gen_server.json

```diff
@@ -12,7 +12,10 @@
       "import { SiemContext } from '../lib/types';"
     ],
     "scalars": {
-      "ToStringArray": "string[]"
+      "ToStringArray": "string[] | string",
+      "ToNumberArray": "number[] | number",
+      "ToDateArray": "string[] | string",
+      "ToBooleanArray": "boolean[] | boolean"
```

Comment:
There's the magic! 🐰🎩 Looks good! Thanks for taking the time _and_ effort to update all these types. Feels much better to be appropriately representing them client-side.